sdk/workflow.go

```diff
@@ -947,6 +948,49 @@ type WorkflowNodeContextDefaultPayloadVCS struct {
 	GitMessage    string `json:"git.message" db:"-"`
 }
 
+// IsWorkflowNodeContextDefaultPayloadVCS checks with several way if the workflow node context has a default vcs payloas
+func IsWorkflowNodeContextDefaultPayloadVCS(i interface{}) bool {
+	_, ok := i.(WorkflowNodeContextDefaultPayloadVCS)
+	if ok {
+		return true
+	}
+
+	dumper := dump.NewDefaultEncoder(nil)
+	dumper.ExtraFields.DetailedMap = false
+	dumper.ExtraFields.DetailedStruct = false
+	dumper.ExtraFields.Len = false
+	dumper.ExtraFields.Type = false
+
+	mI, _ := dumper.ToMap(i)
+	mD, _ := dumper.ToMap(WorkflowNodeContextDefaultPayloadVCS{})
+	var kI, kD []string
+	for k := range mI {
```

Comment:
if len(mI) != len(mD) { 
  return false
}
for k := range mD {
  if ! hasKey(k) {
    return false
  }
}

better than reflect.DeepEqual ?